### PR TITLE
(feat) O3-3203 Deeplink Lab App Orders to Patient Chart Orders

### PR DESCRIPTION
**Pull request by kb019**
## Requirements

- [X] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [X] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [X] My work includes tests or is validated by existing tests.

## Summary
Clicking on a patient name on the Lab App for Tests Ordered and In Progress should deep link  to the patient chart Orders widget

## Related Issue
https://openmrs.atlassian.net/browse/O3-3203

### CODE REVIEW

File: src/config-schema.ts [hunk 1]
```diff
@@ -7,14 +7,24 @@ export const configSchema = {
...
+    },
     _description:
       "The patient chart dashboard to navigate to from the lab app.",
   },
 };
 
 export type Config = {
   laboratoryOrderTypeUuid: string;
-  targetPatientDashboard: string;
+  targetPatientDashboard: ConfigSchema;
```
Comment (pirupius) [inline src/config-schema.ts:29]: I think making this an Object is more correct 
```suggestion
  targetPatientDashboard: Object;
```

File: src/config-schema.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Type } from "@openmrs/esm-framework";
+import { Type, ConfigSchema } from "@openmrs/esm-framework";
```
Comment (pirupius) [inline src/config-schema.ts:1]:
```suggestion
import { Type } from "@openmrs/esm-framework";
```

File: src/components/orders-table/orders-data-table.component.tsx [hunk 3]
```diff
@@ -128,7 +130,11 @@ const OrdersDataTable: React.FC<OrdersDataTableProps> = ({
       ),
       patient: (
         <ConfigurableLink
-          to={`\${openmrsSpaBase}/patient/${order.patient?.uuid}/chart/${targetPatientDashboard}`}
+          to={`\${openmrsSpaBase}/patient/${order.patient?.uuid}/chart/${
```
Comment (pirupius) [inline src/components/orders-table/orders-data-table.component.tsx:133]: Is the `\` necessary for this?
```suggestion
          to={`${openmrsSpaBase}/patient/${order.patient?.uuid}/chart/${
```

Comment (kb019) [reply src/components/orders-table/orders-data-table.component.tsx:133]: Hi  @pirupius , The '\\' at the start is required to stop the interpolation of the dollar sign, so the url becomes "${openmrsSpaBase}\..." ,and then in the configurable link, "${openmrsSpaBase}" is interpolated to "/openmrs/spa".
Thank you @pirupius

File: src/config-schema.ts (hunk 2 above)
Comment (kb019) [reply src/config-schema.ts]: Hi @pirupius , I have updated this Change . Thank you.

File: src/config-schema.ts (hunk 1 above)
Comment (kb019) [reply src/config-schema.ts]: Hi @pirupius , I have updated this Change . Thank you.

**Review by pirupius — APPROVED** [review]

**Pull request merged**